addon/components/simple-bar.js

```diff
@@ -2,6 +2,7 @@ import Component from '@glimmer/component';
 import { tracked } from '@glimmer/tracking';
 import { action } from '@ember/object';
 import SimpleBarCore from 'simplebar-core';
+import 'simplebar-core/dist/simplebar.css';
```

Comment:
This works because this addon depends on EAI v2. The location of the imported css will/might be different, so seems safest to release this as a breaking change?